app/controllers/requests_controller.rb

```diff
@@ -125,6 +125,12 @@ def unsuccessful_action(format, method)
     format.json { render json: @request.errors, status: :unprocessable_entity }
   end
 
+  def prepare_params
+    params[:request][:name] = replace_whitespaces(params[:request][:name])
+    params[:request][:ram_mb] = gb_to_mb(params[:request][:ram_mb].to_i)
```

Comment:
I would add a comment here, to explain why this conversion is necessary.
If you don't know that the view is working in GB and the model is not, this might be confusing